src/pyfaf/faf_rpm.py

```diff
@@ -30,7 +31,7 @@
 __all__ = ["store_rpm_deps", "unpack_rpm_to_tmp"]
 
 
-def parse_evr(evr_string):
+def parse_evr(evr_string) -> Tuple[Optional[str], Optional[str], Optional[str]]:
```

Comment:
As you can see, epoch can also be set to a literal zero, making `Optional[str]` not completely true.